repos/system_upgrade/common/actors/cryptopoliciescheck/actor.py

```diff
@@ -21,4 +22,8 @@ class CryptoPoliciesCheck(Actor):
     tags = (IPUWorkflowTag, ChecksPhaseTag,)
 
     def process(self):
+        # there are no crypto policies in EL 7
+        if int(version.get_target_major_version()) < 9:
```

Comment:
@Jakuje  let's be rather focused on the current system version when the problem is not connected to the target system but to the source system: 

```suggestion
        if version.get_source_major_version() == '7':
```

Reply:
Will fix

Reply:
I will also move it to the library as you did in the other PR we discussed.

Reply:
no. I can't  as the messages are consumed already here. Never mind

Reply:
Resolved.